Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_046.SqlConnectionString.ps1

```diff
@@ -1,5 +1,7 @@
 # This is a well-formed but invalid ADO sql connection string
 server=tcp:some-database-name.database.windows.net,1433;Initial Catalog=catalog-db;Persist Security Info=False;User ID=username;Password=password;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;
+Initial Catalog=catalog-db;Persist Security Info=False;Password=password;MultipleActiveResultSets=False;Encrypt=True;User ID=username;TrustServerCertificate=False;Connection Timeout=30;server=tcp:some-database-name.database.windows.net,1433;
```

Comment:
For my sanity, could you create some examples that are wrapped in XML, like we might see in some config files, or perhaps defined in code?  You don't normally see connection strings in just plain text like this.  I want to verify that quotes/new line characters/etc don't have any unexpected effects.

Reply:
Yes, we have other multiline test patterns that Eddy put together, we should go recreate those. This was a very early rule and the pattern wasn't established yet.

---
In reply to: [569655372](https://github.com/microsoft/sarif-pattern-matcher/pull/160#discussion_r569655372) [](ancestors = 569655372)

Reply:
added more tests and more combinatios, xml style, appsettings style

---
In reply to: [569697107](https://github.com/microsoft/sarif-pattern-matcher/pull/160#discussion_r569697107) [](ancestors = 569697107,569655372)